### ISSUE TITLE
kustomize cfg fmt: Panic on parsing emoji in files.

### ISSUE THREAD

**Issue by matthewhembree** [kubernetes-sigs/kustomize#3895]
**Describe the bug**

A file containing an emoji (such as 👆) will cause `kustomize cfg fmt` to panic. It will not write (net delete) the affected file.

**Files that can reproduce the issue**

```yaml
# kustomization.yaml

namespace: argocd
apiVersion: kustomize.config.k8s.io/v1beta1
kind: Kustomization
resources:
# Hello world.
- https://github.com/argoproj/argo-cd.git//manifests/ha/cluster-install?ref=v2.0.1 # Did you read the line above? 👆
```

**Expected output**

```yaml
# kustomization.yaml

namespace: argocd
apiVersion: kustomize.config.k8s.io/v1beta1
kind: Kustomization
resources:
# Hello world.
- https://github.com/argoproj/argo-cd.git//manifests/ha/cluster-install?ref=v2.0.1 # Did you read the line above? 👆
```

**Actual output**

```
kustomize cfg fmt .
./
panic: runtime error: index out of range [34] with length 32 [recovered]
	panic: runtime error: index out of range [34] with length 32

goroutine 1 [running]:
gopkg.in/yaml%2ev3.handleErr(0xc000562fd8)
	/Users/brew/Library/Caches/Homebrew/go_mod_cache/pkg/mod/gopkg.in/yaml.v3@v3.0.0-20200313102051-9f266ea9e77c/yaml.go:276 +0x8d
panic(0x47a84a0, 0xc0008cc858)
	/usr/local/Cellar/go/1.16.3/libexec/src/runtime/panic.go:965 +0x1b9
gopkg.in/yaml%2ev3.write(0xc00013d180, 0xc0008c5fc0, 0x20, 0x20, 0xc000560d50, 0x4432d01)
	/Users/brew/Library/Caches/Homebrew/go_mod_cache/pkg/mod/gopkg.in/yaml.v3@v3.0.0-20200313102051-9f266ea9e77c/emitterc.go:87 +0x258
gopkg.in/yaml%2ev3.yaml_emitter_write_comment(0xc00013d180, 0xc0008c5fc0, 0x20, 0x20, 0x1)
	/Users/brew/Library/Caches/Homebrew/go_mod_cache/pkg/mod/gopkg.in/yaml.v3@v3.0.0-20200313102051-9f266ea9e77c/emitterc.go:1978 +0x198
gopkg.in/yaml%2ev3.yaml_emitter_process_line_comment(0xc00013d180, 0xc00102abb0)
	/Users/brew/Library/Caches/Homebrew/go_mod_cache/pkg/mod/gopkg.in/yaml.v3@v3.0.0-20200313102051-9f266ea9e77c/emitterc.go:1128 +0x6a
gopkg.in/yaml%2ev3.yaml_emitter_emit_block_sequence_item(0xc00013d180, 0xc00102abb0, 0x44ad201, 0xc00033dc01)
	/Users/brew/Library/Caches/Homebrew/go_mod_cache/pkg/mod/gopkg.in/yaml.v3@v3.0.0-20200313102051-9f266ea9e77c/emitterc.go:759 +0x274
gopkg.in/yaml%2ev3.yaml_emitter_state_machine(0xc00013d180, 0xc00102abb0, 0x47f0d01)
	/Users/brew/Library/Caches/Homebrew/go_mod_cache/pkg/mod/gopkg.in/yaml.v3@v3.0.0-20200313102051-9f266ea9e77c/emitterc.go:291 +0x2d2
gopkg.in/yaml%2ev3.yaml_emitter_emit(0xc00013d180, 0xc00013d3c0, 0xc000c25380)
	/Users/brew/Library/Caches/Homebrew/go_mod_cache/pkg/mod/gopkg.in/yaml.v3@v3.0.0-20200313102051-9f266ea9e77c/emitterc.go:153 +0x14f
gopkg.in/yaml%2ev3.(*encoder).emit(0xc00013d180)
	/Users/brew/Library/Caches/Homebrew/go_mod_cache/pkg/mod/gopkg.in/yaml.v3@v3.0.0-20200313102051-9f266ea9e77c/encode.go:81 +0x39
gopkg.in/yaml%2ev3.(*encoder).emitScalar(0xc00013d180, 0xc00088c7e0, 0xe, 0x0, 0x0, 0x0, 0x0, 0xc0003e5702, 0x4d32e90, 0x0, ...)
	/Users/brew/Library/Caches/Homebrew/go_mod_cache/pkg/mod/gopkg.in/yaml.v3@v3.0.0-20200313102051-9f266ea9e77c/encode.go:417 +0x31a
gopkg.in/yaml%2ev3.(*encoder).node(0xc00013d180, 0xc0001c0e60, 0x0, 0x0)
	/Users/brew/Library/Caches/Homebrew/go_mod_cache/pkg/mod/gopkg.in/yaml.v3@v3.0.0-20200313102051-9f266ea9e77c/encode.go:559 +0x136f
gopkg.in/yaml%2ev3.(*encoder).node(0xc00013d180, 0xc0001ba960, 0x0, 0x0)
	/Users/brew/Library/Caches/Homebrew/go_mod_cache/pkg/mod/gopkg.in/yaml.v3@v3.0.0-20200313102051-9f266ea9e77c/encode.go:478 +0x5fc
gopkg.in/yaml%2ev3.(*encoder).node(0xc00013d180, 0xc0001acaa0, 0x0, 0x0)
	/Users/brew/Library/Caches/Homebrew/go_mod_cache/pkg/mod/gopkg.in/yaml.v3@v3.0.0-20200313102051-9f266ea9e77c/encode.go:512 +0xa6f
gopkg.in/yaml%2ev3.(*encoder).node(0xc00013d180, 0xc0001a68c0, 0x0, 0x0)
	/Users/brew/Library/Caches/Homebrew/go_mod_cache/pkg/mod/gopkg.in/yaml.v3@v3.0.0-20200313102051-9f266ea9e77c/encode.go:463 +0x245
gopkg.in/yaml%2ev3.(*encoder).nodev(0xc00013d180, 0x4773120, 0xc0001a68c0, 0x16)
	/Users/brew/Library/Caches/Homebrew/go_mod_cache/pkg/mod/gopkg.in/yaml.v3@v3.0.0-20200313102051-9f266ea9e77c/encode.go:421 +0x79
gopkg.in/yaml%2ev3.(*encoder).marshalDoc(0xc00013d180, 0x0, 0x0, 0x4773120, 0xc0001a68c0, 0x16)
	/Users/brew/Library/Caches/Homebrew/go_mod_cache/pkg/mod/gopkg.in/yaml.v3@v3.0.0-20200313102051-9f266ea9e77c/encode.go:101 +0x23a
gopkg.in/yaml%2ev3.(*Encoder).Encode(0xc000917028, 0x4773120, 0xc0001a68c0, 0x0, 0x0)
	/Users/brew/Library/Caches/Homebrew/go_mod_cache/pkg/mod/gopkg.in/yaml.v3@v3.0.0-20200313102051-9f266ea9e77c/yaml.go:251 +0xef
sigs.k8s.io/kustomize/kyaml/yaml.String(0xc0001a68c0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0)
	/Users/brew/Library/Caches/Homebrew/go_mod_cache/pkg/mod/sigs.k8s.io/kustomize/kyaml@v0.10.17/yaml/types.go:215 +0x1b3
sigs.k8s.io/kustomize/kyaml/yaml.(*RNode).String(...)
	/Users/brew/Library/Caches/Homebrew/go_mod_cache/pkg/mod/sigs.k8s.io/kustomize/kyaml@v0.10.17/yaml/rnode.go:524
sigs.k8s.io/kustomize/kyaml/kio.ByteWriter.encode(0x48e2220, 0xc000917018, 0x0, 0xc0003e4930, 0x1, 0x1, 0x0, 0x0, 0x0, 0x0, ...)
	/Users/brew/Library/Caches/Homebrew/go_mod_cache/pkg/mod/sigs.k8s.io/kustomize/kyaml@v0.10.17/kio/byteio_writer.go:130 +0x2b8
sigs.k8s.io/kustomize/kyaml/kio.ByteWriter.Write(0x48e2220, 0xc000917018, 0x0, 0xc0003e4930, 0x1, 0x1, 0x0, 0x0, 0x0, 0x0, ...)
	/Users/brew/Library/Caches/Homebrew/go_mod_cache/pkg/mod/sigs.k8s.io/kustomize/kyaml@v0.10.17/kio/byteio_writer.go:87 +0xde6
sigs.k8s.io/kustomize/kyaml/kio.LocalPackageWriter.Write.func1(0xc000917018, 0xc0005636c8, 0xc000130390, 0xc0005635c8, 0xc0005635d8, 0x0, 0x0)
	/Users/brew/Library/Caches/Homebrew/go_mod_cache/pkg/mod/sigs.k8s.io/kustomize/kyaml@v0.10.17/kio/pkgio_writer.go:102 +0x16a
sigs.k8s.io/kustomize/kyaml/kio.LocalPackageWriter.Write(0x0, 0x0, 0x7ffeefbff202, 0x1, 0x0, 0xc0003e4930, 0x1, 0x1, 0xc0000ec000, 0x146, ...)
	/Users/brew/Library/Caches/Homebrew/go_mod_cache/pkg/mod/sigs.k8s.io/kustomize/kyaml@v0.10.17/kio/pkgio_writer.go:106 +0x905
sigs.k8s.io/kustomize/kyaml/kio.(*LocalPackageReadWriter).Write(0xc000034d00, 0xc0000ec000, 0x146, 0x150, 0xc0000ec000, 0x146)
	/Users/brew/Library/Caches/Homebrew/go_mod_cache/pkg/mod/sigs.k8s.io/kustomize/kyaml@v0.10.17/kio/pkgio_reader.go:116 +0x285
sigs.k8s.io/kustomize/kyaml/kio.Pipeline.ExecuteWithCallback(0xc000b7f9e8, 0x1, 0x1, 0xc000b7f9f8, 0x1, 0x1, 0xc000b7f9d8, 0x1, 0x1, 0x0, ...)
	/Users/brew/Library/Caches/Homebrew/go_mod_cache/pkg/mod/sigs.k8s.io/kustomize/kyaml@v0.10.17/kio/kio.go:131 +0x251
sigs.k8s.io/kustomize/kyaml/kio.Pipeline.Execute(...)
	/Users/brew/Library/Caches/Homebrew/go_mod_cache/pkg/mod/sigs.k8s.io/kustomize/kyaml@v0.10.17/kio/kio.go:93
sigs.k8s.io/kustomize/cmd/config/internal/commands.(*FmtRunner).ExecuteCmd(0xc000514e20, 0x48e2220, 0xc000010018, 0x7ffeefbff202, 0x1, 0x1, 0x1)
	/Users/brew/Library/Caches/Homebrew/go_mod_cache/pkg/mod/sigs.k8s.io/kustomize/cmd/config@v0.9.10/internal/commands/fmt.go:105 +0x24a
sigs.k8s.io/kustomize/cmd/config/runner.ExecuteCmdOnPkgs.processPkg(0x7ffeefbff202, 0x1, 0x0, 0x48e24a0, 0xc000514e20, 0x48e2220, 0xc000010018, 0x0, 0x7ffeefbff202, 0x1, ...)
	/Users/brew/Library/Caches/Homebrew/go_mod_cache/pkg/mod/sigs.k8s.io/kustomize/cmd/config@v0.9.10/runner/runner.go:78 +0x79
sigs.k8s.io/kustomize/cmd/config/runner.ExecuteCmdOnPkgs.Execute(0x7ffeefbff202, 0x1, 0x0, 0x48e24a0, 0xc000514e20, 0x48e2220, 0xc000010018, 0x0, 0x4170d25, 0x4736480)
	/Users/brew/Library/Caches/Homebrew/go_mod_cache/pkg/mod/sigs.k8s.io/kustomize/cmd/config@v0.9.10/runner/runner.go:61 +0x225
sigs.k8s.io/kustomize/cmd/config/internal/commands.(*FmtRunner).runE(0xc000514e20, 0xc00011f340, 0xc000308c80, 0x1, 0x1, 0x0, 0x0)
	/Users/brew/Library/Caches/Homebrew/go_mod_cache/pkg/mod/sigs.k8s.io/kustomize/cmd/config@v0.9.10/internal/commands/fmt.go:91 +0x16d
github.com/spf13/cobra.(*Command).execute(0xc00011f340, 0xc000308c50, 0x1, 0x1, 0xc00011f340, 0xc000308c50)
	/Users/brew/Library/Caches/Homebrew/go_mod_cache/pkg/mod/github.com/spf13/cobra@v1.0.0/command.go:842 +0x472
github.com/spf13/cobra.(*Command).ExecuteC(0xc0000de2c0, 0x0, 0xffffffff, 0xc0000320b8)
	/Users/brew/Library/Caches/Homebrew/go_mod_cache/pkg/mod/github.com/spf13/cobra@v1.0.0/command.go:950 +0x375
github.com/spf13/cobra.(*Command).Execute(...)
	/Users/brew/Library/Caches/Homebrew/go_mod_cache/pkg/mod/github.com/spf13/cobra@v1.0.0/command.go:887
main.main()
	/private/tmp/kustomize-20210415-81672-oe6iwg/kustomize/main.go:14 +0x2a
```

```yaml
# empty file
```

**Kustomize version**

{Version:kustomize/v4.1.2 GitCommit:a5914abad89e0b18129eaf1acc784f9fe7d21439 BuildDate:2021-04-15T23:23:37+01:00 GoOs:darwin GoArch:amd64}

**Platform**

macOS

**Additional context**

`kustomize build` works fine.

I'm not saying I *need* emoji support. I'm just opening a bug noting the behavior. And it's ungraceful error handling.

Thanks!

**Comment by Shell32-Natsu** [comment]
Could you try with non-ascii characters, like CJK? Want to confirm is this a emoji specific issue or a utf-8 issue.

**Comment by matthewhembree** [comment]
> Could you try with non-ascii characters, like CJK? Want to confirm is this an emoji specific issue or a utf-8 issue.

@Shell32-Natsu 

This fails too:
```yaml
namespace: argocd
apiVersion: kustomize.config.k8s.io/v1beta1
kind: Kustomization
resources:
# こんにちは
- https://github.com/argoproj/argo-cd.git/manifests/ha/cluster-install?ref=v2.0.1 # Did you read the line above?
```

```
kustomize cfg fmt ./argocd
./argocd/
panic: runtime error: index out of range [7] with length 7 [recovered]
	panic: runtime error: index out of range [7] with length 7

goroutine 1 [running]:
gopkg.in/yaml%2ev3.handleErr(0xc0008fcfd8)
	/Users/brew/Library/Caches/Homebrew/go_mod_cache/pkg/mod/gopkg.in/yaml.v3@v3.0.0-20200313102051-9f266ea9e77c/yaml.go:276 +0x8d
panic(0x47a84a0, 0xc0005298f0)
	/usr/local/Cellar/go/1.16.3/libexec/src/runtime/panic.go:965 +0x1b9
gopkg.in/yaml%2ev3.write(0xc00020a700, 0xc000700df8, 0x7, 0x8, 0xc0008fad50, 0xc0008fad01)
	/Users/brew/Library/Caches/Homebrew/go_mod_cache/pkg/mod/gopkg.in/yaml.v3@v3.0.0-20200313102051-9f266ea9e77c/emitterc.go:90 +0x245
gopkg.in/yaml%2ev3.yaml_emitter_write_comment(0xc00020a700, 0xc000700df8, 0x7, 0x8, 0x50)
	/Users/brew/Library/Caches/Homebrew/go_mod_cache/pkg/mod/gopkg.in/yaml.v3@v3.0.0-20200313102051-9f266ea9e77c/emitterc.go:1978 +0x198
gopkg.in/yaml%2ev3.yaml_emitter_process_head_comment(0xc00020a700, 0x101000000000000)
	/Users/brew/Library/Caches/Homebrew/go_mod_cache/pkg/mod/gopkg.in/yaml.v3@v3.0.0-20200313102051-9f266ea9e77c/emitterc.go:1111 +0x88
gopkg.in/yaml%2ev3.yaml_emitter_emit_block_sequence_item(0xc00020a700, 0xc000179eb0, 0x5500101, 0xc000880a01)
	/Users/brew/Library/Caches/Homebrew/go_mod_cache/pkg/mod/gopkg.in/yaml.v3@v3.0.0-20200313102051-9f266ea9e77c/emitterc.go:746 +0x198
gopkg.in/yaml%2ev3.yaml_emitter_state_machine(0xc00020a700, 0xc000179eb0, 0x47f0d01)
	/Users/brew/Library/Caches/Homebrew/go_mod_cache/pkg/mod/gopkg.in/yaml.v3@v3.0.0-20200313102051-9f266ea9e77c/emitterc.go:291 +0x2d2
gopkg.in/yaml%2ev3.yaml_emitter_emit(0xc00020a700, 0xc00020a940, 0xc000700e20)
	/Users/brew/Library/Caches/Homebrew/go_mod_cache/pkg/mod/gopkg.in/yaml.v3@v3.0.0-20200313102051-9f266ea9e77c/emitterc.go:153 +0x14f
gopkg.in/yaml%2ev3.(*encoder).emit(0xc00020a700)
	/Users/brew/Library/Caches/Homebrew/go_mod_cache/pkg/mod/gopkg.in/yaml.v3@v3.0.0-20200313102051-9f266ea9e77c/encode.go:81 +0x39
gopkg.in/yaml%2ev3.(*encoder).emitScalar(0xc00020a700, 0xc000a05c80, 0xe, 0x0, 0x0, 0x0, 0x0, 0xc000469d02, 0x4d32e90, 0x0, ...)
	/Users/brew/Library/Caches/Homebrew/go_mod_cache/pkg/mod/gopkg.in/yaml.v3@v3.0.0-20200313102051-9f266ea9e77c/encode.go:417 +0x31a
gopkg.in/yaml%2ev3.(*encoder).node(0xc00020a700, 0xc0008f5e00, 0x0, 0x0)
	/Users/brew/Library/Caches/Homebrew/go_mod_cache/pkg/mod/gopkg.in/yaml.v3@v3.0.0-20200313102051-9f266ea9e77c/encode.go:559 +0x136f
gopkg.in/yaml%2ev3.(*encoder).node(0xc00020a700, 0xc0008f5cc0, 0x0, 0x0)
	/Users/brew/Library/Caches/Homebrew/go_mod_cache/pkg/mod/gopkg.in/yaml.v3@v3.0.0-20200313102051-9f266ea9e77c/encode.go:478 +0x5fc
gopkg.in/yaml%2ev3.(*encoder).node(0xc00020a700, 0xc0008f57c0, 0x0, 0x0)
	/Users/brew/Library/Caches/Homebrew/go_mod_cache/pkg/mod/gopkg.in/yaml.v3@v3.0.0-20200313102051-9f266ea9e77c/encode.go:512 +0xa6f
gopkg.in/yaml%2ev3.(*encoder).node(0xc00020a700, 0xc0008f5680, 0x0, 0x0)
	/Users/brew/Library/Caches/Homebrew/go_mod_cache/pkg/mod/gopkg.in/yaml.v3@v3.0.0-20200313102051-9f266ea9e77c/encode.go:463 +0x245
gopkg.in/yaml%2ev3.(*encoder).nodev(0xc00020a700, 0x4773120, 0xc0008f5680, 0x16)
	/Users/brew/Library/Caches/Homebrew/go_mod_cache/pkg/mod/gopkg.in/yaml.v3@v3.0.0-20200313102051-9f266ea9e77c/encode.go:421 +0x79
gopkg.in/yaml%2ev3.(*encoder).marshalDoc(0xc00020a700, 0x0, 0x0, 0x4773120, 0xc0008f5680, 0x16)
	/Users/brew/Library/Caches/Homebrew/go_mod_cache/pkg/mod/gopkg.in/yaml.v3@v3.0.0-20200313102051-9f266ea9e77c/encode.go:101 +0x23a
gopkg.in/yaml%2ev3.(*Encoder).Encode(0xc0006900c0, 0x4773120, 0xc0008f5680, 0x0, 0x0)
	/Users/brew/Library/Caches/Homebrew/go_mod_cache/pkg/mod/gopkg.in/yaml.v3@v3.0.0-20200313102051-9f266ea9e77c/yaml.go:251 +0xef
sigs.k8s.io/kustomize/kyaml/yaml.String(0xc0008f5680, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0)
	/Users/brew/Library/Caches/Homebrew/go_mod_cache/pkg/mod/sigs.k8s.io/kustomize/kyaml@v0.10.17/yaml/types.go:215 +0x1b3
sigs.k8s.io/kustomize/kyaml/yaml.(*RNode).String(...)
	/Users/brew/Library/Caches/Homebrew/go_mod_cache/pkg/mod/sigs.k8s.io/kustomize/kyaml@v0.10.17/yaml/rnode.go:524
sigs.k8s.io/kustomize/kyaml/kio.ByteWriter.encode(0x48e2220, 0xc0006900b0, 0x0, 0xc000a09c40, 0x1, 0x1, 0x0, 0x0, 0x0, 0x0, ...)
	/Users/brew/Library/Caches/Homebrew/go_mod_cache/pkg/mod/sigs.k8s.io/kustomize/kyaml@v0.10.17/kio/byteio_writer.go:130 +0x2b8
sigs.k8s.io/kustomize/kyaml/kio.ByteWriter.Write(0x48e2220, 0xc0006900b0, 0x0, 0xc000a09c40, 0x1, 0x1, 0x0, 0x0, 0x0, 0x0, ...)
	/Users/brew/Library/Caches/Homebrew/go_mod_cache/pkg/mod/sigs.k8s.io/kustomize/kyaml@v0.10.17/kio/byteio_writer.go:87 +0xde6
sigs.k8s.io/kustomize/kyaml/kio.LocalPackageWriter.Write.func1(0xc0006900b0, 0xc0008fd6c8, 0xc000200e10, 0xc0008fd5c8, 0xc0008fd5d8, 0x0, 0x0)
	/Users/brew/Library/Caches/Homebrew/go_mod_cache/pkg/mod/sigs.k8s.io/kustomize/kyaml@v0.10.17/kio/pkgio_writer.go:102 +0x16a
sigs.k8s.io/kustomize/kyaml/kio.LocalPackageWriter.Write(0x0, 0x0, 0x7ffeefbff202, 0x8, 0x0, 0xc000a09c40, 0x1, 0x1, 0xc000512400, 0x10f, ...)
	/Users/brew/Library/Caches/Homebrew/go_mod_cache/pkg/mod/sigs.k8s.io/kustomize/kyaml@v0.10.17/kio/pkgio_writer.go:106 +0x905
sigs.k8s.io/kustomize/kyaml/kio.(*LocalPackageReadWriter).Write(0xc000441080, 0xc000512400, 0x10f, 0x120, 0xc000512400, 0x10f)
	/Users/brew/Library/Caches/Homebrew/go_mod_cache/pkg/mod/sigs.k8s.io/kustomize/kyaml@v0.10.17/kio/pkgio_reader.go:116 +0x285
sigs.k8s.io/kustomize/kyaml/kio.Pipeline.ExecuteWithCallback(0xc0008fd9e8, 0x1, 0x1, 0xc0008fd9f8, 0x1, 0x1, 0xc0008fd9d8, 0x1, 0x1, 0x0, ...)
	/Users/brew/Library/Caches/Homebrew/go_mod_cache/pkg/mod/sigs.k8s.io/kustomize/kyaml@v0.10.17/kio/kio.go:131 +0x251
sigs.k8s.io/kustomize/kyaml/kio.Pipeline.Execute(...)
	/Users/brew/Library/Caches/Homebrew/go_mod_cache/pkg/mod/sigs.k8s.io/kustomize/kyaml@v0.10.17/kio/kio.go:93
sigs.k8s.io/kustomize/cmd/config/internal/commands.(*FmtRunner).ExecuteCmd(0xc0005252e0, 0x48e2220, 0xc000198008, 0x7ffeefbff202, 0x8, 0x1, 0x1)
	/Users/brew/Library/Caches/Homebrew/go_mod_cache/pkg/mod/sigs.k8s.io/kustomize/cmd/config@v0.9.10/internal/commands/fmt.go:105 +0x24a
sigs.k8s.io/kustomize/cmd/config/runner.ExecuteCmdOnPkgs.processPkg(0x7ffeefbff202, 0x8, 0x0, 0x48e24a0, 0xc0005252e0, 0x48e2220, 0xc000198008, 0x0, 0x7ffeefbff202, 0x8, ...)
	/Users/brew/Library/Caches/Homebrew/go_mod_cache/pkg/mod/sigs.k8s.io/kustomize/cmd/config@v0.9.10/runner/runner.go:78 +0x79
sigs.k8s.io/kustomize/cmd/config/runner.ExecuteCmdOnPkgs.Execute(0x7ffeefbff202, 0x8, 0x0, 0x48e24a0, 0xc0005252e0, 0x48e2220, 0xc000198008, 0x0, 0x4170d25, 0x4736480)
	/Users/brew/Library/Caches/Homebrew/go_mod_cache/pkg/mod/sigs.k8s.io/kustomize/cmd/config@v0.9.10/runner/runner.go:61 +0x225
sigs.k8s.io/kustomize/cmd/config/internal/commands.(*FmtRunner).runE(0xc0005252e0, 0xc0007b9340, 0xc0003ca1a0, 0x1, 0x1, 0x0, 0x0)
	/Users/brew/Library/Caches/Homebrew/go_mod_cache/pkg/mod/sigs.k8s.io/kustomize/cmd/config@v0.9.10/internal/commands/fmt.go:91 +0x16d
github.com/spf13/cobra.(*Command).execute(0xc0007b9340, 0xc0003ca170, 0x1, 0x1, 0xc0007b9340, 0xc0003ca170)
	/Users/brew/Library/Caches/Homebrew/go_mod_cache/pkg/mod/github.com/spf13/cobra@v1.0.0/command.go:842 +0x472
github.com/spf13/cobra.(*Command).ExecuteC(0xc0001dc2c0, 0x0, 0xffffffff, 0xc00010e058)
	/Users/brew/Library/Caches/Homebrew/go_mod_cache/pkg/mod/github.com/spf13/cobra@v1.0.0/command.go:950 +0x375
github.com/spf13/cobra.(*Command).Execute(...)
	/Users/brew/Library/Caches/Homebrew/go_mod_cache/pkg/mod/github.com/spf13/cobra@v1.0.0/command.go:887
main.main()
	/private/tmp/kustomize-20210415-81672-oe6iwg/kustomize/main.go:14 +0x2a
```

**Comment by brianpursley** [comment]
@matthewhembree, I reproduced this bug using version 4.1.2, but it seems to be fixed in version 4.1.3 (both examples you provided).

Can you try with kustomize version 4.1.3?

**Comment by matthewhembree** [comment]
@brianpursley I confirm that the issue has been resolved in 4.1.3 for both cases. 👍
